### PR TITLE
BD-2865 New WhatsApp article for multiple WABAs

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #**BD-2865**

#### Is this change associated with a Braze feature/product release?
- [x] Yes (**Insert Feature Release Date Here**)
- [ ] No

### CODE REVIEW

File: _docs/_user_guide/message_building_by_channel/whatsapp/overview/multiple_subscription_groups.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+- Status 
+- Quality rating 
+- Messaging limit 
+
+### Adding a subscription group and phone number
+
+You can add up to 20 subscription groups (and sending phone numbers) per WhatsApp Business account. To add a subscription group and phone number:
+
+1. Go to **Technology Partners** > **WhatsApp** and select **Add Subscription Group and Number**.![WhatsApp Messaging Integration section with options to add a business account or add a subscription group and number.][1]<br>
+2. Go through the sign up workflow. In the **Select your WhatsApp Business Account** step, select your existing WhatsApp Business account and add a new phone number. This number must follow all the requirements of any WhatsApp phone number, including not being registered to any other WhatsApp accounts.
```
Comment (lydia-xie) [inline _docs/_user_guide/message_building_by_channel/whatsapp/overview/multiple_subscription_groups.md:48]:
```suggestion
2. Go through the sign-up workflow. In the **Select your WhatsApp Business Account** step, select your existing WhatsApp Business account and add a new phone number. This number must follow all the requirements of any WhatsApp phone number, including not being registered to any other WhatsApp accounts.
```

File: _docs/_user_guide/message_building_by_channel/whatsapp/overview/multiple_subscription_groups.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+Having multiple WhatsApp Business accounts is useful if you want to send WhatsApp messages to users in a Braze workspace that has multiple brands. This is because each business account operates separately within WhatsApp and has its own phone number, message template, and quality rating.
+
+Business accounts that are nested within the same Meta Business Manager will also share user access permission management and catalogs (not yet supported on Braze).
+
+### Adding a WhatsApp Business account
+
+You can add up to 10 WhatsApp Business accounts per workspace. To add an account:
+
+1. Go **Technology Partners** > **WhatsApp** and select **Add WhatsApp Business Account**. ![WhatsApp Messaging Integration section with options to add a business account or add a subscription group and number.][1]<br>
+2. Go through the sign up workflow. In the **Select your WhatsApp Business Account** step, select **Create a new WhatsApp Business Account**.
```
Comment (lydia-xie) [inline _docs/_user_guide/message_building_by_channel/whatsapp/overview/multiple_subscription_groups.md:30]:
```suggestion
2. Go through the sign-up workflow. In the **Select your WhatsApp Business Account** step, select **Create a new WhatsApp Business Account**.
```

File: _docs/_user_guide/message_building_by_channel/whatsapp/overview/multiple_subscription_groups.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+You can add up to 20 subscription groups (and sending phone numbers) per WhatsApp Business account. To add a subscription group and phone number:
+
+1. Go to **Technology Partners** > **WhatsApp** and select **Add Subscription Group and Number**.![WhatsApp Messaging Integration section with options to add a business account or add a subscription group and number.][1]<br>
+2. Go through the sign up workflow. In the **Select your WhatsApp Business Account** step, select your existing WhatsApp Business account and add a new phone number. This number must follow all the requirements of any WhatsApp phone number, including not being registered to any other WhatsApp accounts.
+
+### Removing a subscription group and phone number 
+
+1. Go **Audience** > **Subscriptions** and archive the subscription group.
+2. Go to your Meta manager and delete the phone number.
```
Comment (lydia-xie) [inline _docs/_user_guide/message_building_by_channel/whatsapp/overview/multiple_subscription_groups.md:53]:
```suggestion
2. Go to your Meta manager and delete the phone number.
```
Should "Meta manager" be "Meta Business Manager"? If not, disregard this comment! :)

**Review by lydia-xie — APPROVED** [review]
lgtm! left a comment for your review!

File: _docs/_user_guide/message_building_by_channel/whatsapp/overview/multiple_subscription_groups.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+You can add up to 20 subscription groups (and sending phone numbers) per WhatsApp Business account. To add a subscription group and phone number:
+
+1. Go to **Technology Partners** > **WhatsApp** and select **Add Subscription Group and Number**.![WhatsApp Messaging Integration section with options to add a business account or add a subscription group and number.][1]<br>
+2. Go through the sign-up workflow. In the **Select your WhatsApp Business Account** step, select your existing WhatsApp Business account and add a new phone number. This number must follow all the requirements of any WhatsApp phone number, including not being registered to any other WhatsApp accounts.
+
+### Removing a subscription group and phone number 
+
+1. Go **Audience** > **Subscriptions** and archive the subscription group.
+2. Go to your Meta manager and delete the phone number.
```
Comment (rachel-feinberg) [inline _docs/_user_guide/message_building_by_channel/whatsapp/overview/multiple_subscription_groups.md:53]:
```suggestion
2. Go to your Meta Business manager and delete the phone number.
```

**Pull request merged**